mne/event.py

```diff
@@ -209,7 +210,12 @@ def read_events(filename, include=None, exclude=None, mask=0):
         the exclude parameter is ignored.
     mask : int | None
         The value of the digital mask to apply to the stim channel values.
-        The default value is 0. ``None`` skips masking.
+        The default value is None. If None, trigger masking is skipped.
+    mask_type: 'and' | 'not_and'
+        The type of operation between the mask and the trigger.
+        Choose 'and' for MNE-C masking behavior.
```

Comment:
I don't find the explanation of `mask` and `mask_type` arguments here very clear. They are explained clearly in docstring of `find_events` - maybe it should be linked/referenced here somehow?